### PR TITLE
Feat/xcm transact

### PR DESCRIPTION
**Pull request by Dinonard**
Add chapter about remote transact to XCM builders part.

### CODE REVIEW

File: docs/xcm/building-with-xcm/xc-remote-transact.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+![2-sign-no-submit](img/remote-transact/002_unsigned_transaction.png)
+
+**Step 3** is to press **Sign** and store the *Signed transaction* data for further use.
+![3-signed-tx-data](img/remote-transact/003_non_signed_tx_data.png)
+
+**Step 4** is to open the **RPC** handle under **Developer** and select `payment->QueryInfo`. Copy the stored `Signed transaction` data into prompt and submit the RPC call. Reply will show how much this transaction weights and how much does it cost to execute it.
+In this particular case, we can see it costs **941_000_000** units of weight to execute.
+![4-query-sig-tx-info](img/remote-transact/004_rpc_query_info_weight_transact_call.png)
+
+**Step 5** is a bit hacky - we need to know how much executing each XCM instruction costs in the destination chain. For `Astar` and `Shiden` it's `1_000_000_000` units of weight per instruction - this is more or less used by every parachain, there's no guarantee however. Knowing this, we can take the transact call weight and sum it up with 4 times weight of a single XCM instruction.
```
Comment (0x7CFE) [inline docs/xcm/building-with-xcm/xc-remote-transact.md:109]: This sentence is kinda hard to parse. Maybe rephrase it like "…how much does it cost to execute XCM instruction on the destination chain"?

**Review by 0x7CFE — APPROVED** [review]

File: docs/xcm/building-with-xcm/xc-remote-transact.md (hunk 1 above)
Comment (Dinonard) [reply docs/xcm/building-with-xcm/xc-remote-transact.md]: I couldn't phrase it as a question but I tried to rephrase it a bit so it's hopefully more clear.

Comment (0x7CFE) [reply docs/xcm/building-with-xcm/xc-remote-transact.md]: I didn't meant to make it a question. The question mark was for my proposal, not the content.

**Review by 0x7CFE — APPROVED** [review]

**Review by fiexer — APPROVED** [review]
